build/codegen/codegen.sh

```diff
@@ -32,7 +32,10 @@ bash ${codegendir}/generate-groups.sh \
     all \
     github.com/rook/rook/pkg/client \
     github.com/rook/rook/pkg/apis \
-    "${GROUP_VERSIONS}"
+    "${GROUP_VERSIONS}" \
+    --output-base "${scriptdir}/../../vendor" \
+    --go-header-file "${scriptdir}/header.txt"
+cp -r "${scriptdir}/../../vendor/github.com/rook/rook/pkg/" "${scriptdir}/../../pkg/"
```

Comment:
I'm not sure about this, this looks a bit intrusive. I'd rather have a doc fix instead if that's the only way to fix it.

Reply:
Is this still necessary since we use `--go-header-file` from the build dir?

Reply:
Yes. Without `--go-header-file`, `code-generator` searches `$GOPATH/src/k8s.io/code-generator` to find boilerplate by default. So it's necessary to specify the template file.
(code-generator doesn't support GoModules so much..)

Reply:
I'll add some docs for that.

Reply:
No, I meant is the `cp -r "${scriptdir}/../../vendor/github.com/rook...` still required now? I don't think so.

Reply:
Oh, sorry. 
`cp -r "${scriptdir}/../../vendor/github.com/rook...` and `--output-base` option doesn't related to https://github.com/rook/rook/issues/5083 directly.  It's required when we work and execute `codegen.sh` outside $GOPATH.  If it's not necessary now, I'll remove it.

Reply:
Since `code-generator` generates files with `github.com/rook/rook` path prefix, we need to copy  them to our pkg directory.